modules/markup/renderer.go

```diff
@@ -163,6 +177,23 @@ type nopCloser struct {
 
 func (nopCloser) Close() error { return nil }
 
+func renderIFrame(ctx *RenderContext, output io.Writer) error {
+	_, err := io.WriteString(output, fmt.Sprintf(`
+<iframe src="%s%s/%s/render/%s/%s"
```

Comment:
```suggestion
<iframe src="%s/%s/%s/render/%s/%s"
```

AppSubURL doesn't have the trailing slash, so here the slash is needed.